src/player.cpp

```diff
@@ -4160,14 +4160,14 @@ void player::pause()
     search_surroundings();
 }
 
-void player::shout( std::string msg )
+void player::shout( std::string msg, const std::string &verbal_shout_id )
 {
-    int base = 10;
+    int base = 85;
```

Comment:
> The problem is, it solves those problems only if you know what to search for and where.

Of course. You'd shout out to a survivor if you suspect he could be within your shout range e.g. he recently ran out of your line of sight. What else were you thinking?

> Quest givers can still run off, you just have better ability to catch them if you know they are in range (which you don't know if LoS limits it)

Again, of course. You aren't supposed to _know_ whether they are in shouting range if you can't see them... At best, you may suspect they are within shouting range because they recently left your line of sight. The idea is to move away from super vision that allows you to see the location of all NPCs on the overmap.

> Does nothing to help with NPCs charging into zeds

Sure it does. Call the NPC before it engages monsters. If you want a shout to command an NPC to disengage a fight, that's out of scope of this change, but it is easy enough to add it in addition to this shout once this PR is merged.

> You'd have to "scan" for NPCs by shouting periodically to find hidden NPCs if they weren't shown on the map.

The problem isn't how to "scan" for NPCs. The problem is how to locate an NPC you suspect is nearby (e.g. because he recently ran out of line of sight). If you expect the NPC is hidden in a nearby building, you can shout to the NPC to call him. You're moving the goal post.

> And this would work better if you could do it without getting an entire horde every time you do that

No kidding. But that's not realistic of a shout. A 100% safe method of finding NPCs isn't the objective here. See https://github.com/CleverRaven/Cataclysm-DDA/issues/16454 for a safer option.

> For it to actually help with those issues, the NPCs would have to shout back.

Why? If the NPC comes when you call for him, why would the NPC need to call back? Also, that would be some lame AI if it called back in response to your call as it could draw more hordes. If you want to implement an ACK protocol in shouts, that's also possible, but not the objective here.